### PR TITLE
Add an area-owners.json file. Remove unused area. Format MD table.

### PR DESCRIPTION
**Pull request by jeffhandley**
The area-owners.json file was generated by parsing the area-owners.md file via a script here:
https://github.com/terrajobst/issuesof.net/pull/12/files?w=1#diff-1a4355995369b2c5ad8b3e1a74cad823ba8fed8fa0172c3f62d5c3ff336c3ed0

That script expands GitHub team members into the list of owners. And terrajobst/issuesof.net#12 will add support for loading the area-owners.json file to override the area-owners.md file so that issuesof.net queries involving `area-owner:` will work as expected even where we're using GitHub teams.

### CODE REVIEW

**Comment by danmoseley** [comment]
Should the script get checked in here if both input and output are checked in here? I guess it is not just for this repo.

**Review by danmoseley — APPROVED** [review]

**Comment by jeffschwMSFT** [comment]
@jeffhandley what is our process for keeping area-owners.json up to date?  area-owners.md gets updates a few times a year (bi-monthly?).  Is it OK if area-owners.json lags or is it the expectation that future updates update both?  (by hand or by tool?)

File: docs/area-owners.json [hunk 1]
```diff
@@ -0,0 +1,1351 @@
...
+        "333fred",
+        "cston"
+      ],
+      "label": "area-Microsoft.CSharp"
+    },
+    {
+      "lead": "ericstj",
+      "owners": null,
+      "label": "area-Microsoft.Extensions"
+    },
```
Comment (teo-tsirpanis) [inline docs/area-owners.json:429]: The area was removed from the MD file.

```suggestion
```

Comment (jeffhandley) [reply docs/area-owners.json]: Good catch; thanks!

**Comment by jeffhandley** [comment]
If the JSON and MD drift a little bit, it's just going to affect the issuesof.net `area-owner` searches for now. @terrajobst and I have discussed trying to get the generation/sync process moved into a GitHub Action or another piece of automation. We haven't fully figured it out yet though. That's also why I didn't bother checking the script in here.

For now, this gets us a stopgap while in the 7.0 endgame and folks want to be able to use the area owner searches.

**Pull request merged**